cli/compose/convert/service.go

```diff
@@ -435,3 +447,20 @@ func convertDeployMode(mode string, replicas *uint64) (swarm.ServiceMode, error)
 	}
 	return serviceMode, nil
 }
+
+func convertCredentialSpec(source string) (*swarm.CredentialSpec, error) {
+	if source == "" {
+		return nil, nil
+	}
+
+	credentialSpec := swarm.CredentialSpec{}
+	switch {
+	case strings.HasPrefix(source, "file://"):
```

Comment:
I think it's fine to leave it as-is:

 1. it makes compose and cli more similar
 2. it's simpler to add additional schemes (I'm looking forward to `config`, for example)